mkdocs.yml

```diff
@@ -119,7 +119,9 @@ nav:
   - Get help: getting-started/help.md
 - Tutorials:
   - Load test HTTP with SLOs: tutorials/load-test-http.md
```

Comment:
I feel the first link `load-test-http.md` can be removed and the corresponding file can be deleted. It is not really informative (the tabs provide all the information needed for nav).